Sources/tinys3/AWSRequest.swift

```diff
@@ -77,98 +77,6 @@ struct AWSRequest {
     public var headers: HttpHeaders {
         HttpHeaders().adding(request.allHTTPHeaderFields!)
     }
-
-    static func headRequest(
```

Comment:
Moved all those existing `static func` convenience constructors at the end of the file — within the `extension` that already existed for `// MARK: Convenience Initializers`… but was mostly empty so far.